src/meta-router/meta-router.ts

```diff
@@ -99,7 +99,6 @@ export class MetaRouter implements Router {
           erc20Permit as Index,
           amountInInputToken,
           inputTokenAddress,
-          options,
```

Comment:
how would you pass zeroEx slippage options now?

Reply:
We don't  use this function, I am not sure that we need it

Reply:
will be deleted from the MetaRouter